index.html

```diff
@@ -37,6 +37,128 @@ <h1 class="name">
 			<div class="bottom_geometry"></div>
 		</section>
 
+		<section class="project_section">
+			<h1 class="projects_label">
+				Projects
+			</h1>
+			<div class="project_card">
+				<div class="project_image"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
+					</div>
+					<div class="see_project">
+						<p class="see_project_text">
+							See this project
+						</p>
+						<div class="arrow_forward"></div>
+					</div>
+				</div>
+			</div>
+			<div class="project_card">
+				<div class="project_image_1"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
+					</div>
+					<div class="see_project">
+						<p class="see_project_text">
+							See this project
+						</p>
+						<div class="arrow_forward"></div>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="about_section">
+			<h1 class="about_title">
+				About
+			</h1>
+
+			<p class="about_paragraph">
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+				dolore magna
+				aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+			</p>
+
+			<div class="resume_button">
+				<p class="get_cv_text">
+					Get my CV
+				</p>
+				<div class="download_icon"></div>
+			</div>
+
+			<div class="my_langauges_card">
+				<h1 class="langauge_label">
+					Languages
+				</h1>
+				<ul>
+					<li class="my_langaues_text_style">JavaScript(ES5/ES6)</li>
+					<li class="my_langaues_text_style">Ruby</li>
+					<li class="my_langaues_text_style">HTML</li>
+					<li class="my_langaues_text_style">CSS</li>
+				</ul>
+			</div>
+			<div class="my_frameworks_card">
+				<h1 class="langauge_label">
+					Frameworks
+				</h1>
+				<ul>
+					<li class="my_langaues_text_style">Bootstrap</li>
+					<li class="my_langaues_text_style">Ruby on rails </li>
+					<li class="my_langaues_text_style">RSpec</li>
+					<li class="my_langaues_text_style">Capibara</li>
+					<li class="my_langaues_text_style">Selenium</li>
+				</ul>
```

Comment:
- [ ] The list of languages is not the center of the area. You can see in the original design it is near to left. Please change it. Apply it to all of them.

![image](https://user-images.githubusercontent.com/38635158/148406051-11e54858-da76-4d4d-8b9d-c904afc5d233.png)
___
### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/148406107-147d1e71-ce9b-4605-ae04-185ee9eb5ff2.png)